web/src/components/PreflightResultPage.tsx

```diff
@@ -181,56 +181,62 @@ function PreflightResultPage(props: Props) {
         </div>
       </div>
 
-      {props.fromLicenseFlow && (
-        <div className="flex-auto flex justifyContent--flexEnd u-marginBottom--15">
-          {!preflightCheck?.showPreflightCheckPending && (
-            <div>
-              <button
-                type="button"
-                className="btn primary blue"
-                disabled={preflightCheck?.showDeploymentBlocked}
-                onClick={() =>
-                  preflightCheck?.shouldShowConfirmContinueWithFailedPreflights
-                    ? setShowContinueWithFailedPreflightsModal(true)
-                    : deployKotsDownstream({
-                        continueWithFailedPreflights: true,
-                      })
-                }
-              >
+      <div className="flex justifyContent--flexEnd tw-gap-5">
+        {preflightCheck?.shouldShowRerunPreflight && (
```

Comment:
so basically we're just removing the `fromLIcenseFlow` from consideration when displaying the rerun button?

Reply:
yep! there might be more changes later on